x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel_wrapper.tsx

```diff
@@ -199,10 +237,30 @@ export function WorkspacePanelWrapper({
         className={classNames('lnsWorkspacePanelWrapper stretch-for-sharing', {
           'lnsWorkspacePanelWrapper--fullscreen': isFullscreen,
         })}
+        css={{ height: '100%' }}
         color="transparent"
       >
-        <WorkspaceTitle />
-        {children}
+        <EuiFlexGroup
+          gutterSize="none"
+          alignItems="center"
+          justifyContent="center"
+          direction="column"
+          css={{ height: '100%' }}
```

Comment:
nit: could we instead use the emotion syntax? It's more performant this way
```suggestion
          css={css`
            height: 100%;
          `}
```

Reply:
Actually, I see now that it uses emotion either way. See [the docs](https://emotion.sh/docs/css-prop#object-styles)

Reply:
nice! TIL 😁